### DISCUSSION TITLE
Installing Symfony 6.0.10 : 2 errors in the symfony profiler

### DISCUSSION THREAD

**Discussion by eved42** [symfony/symfony#46887] [Q&A] [answered]
### Symfony version(s) affected

6.0.10

### Description

I created a new Symfony 6.0.10 project locally on my PC (Windows 11).

Command : 
```
symfony new my_project --version="6.0.*" --webapp
symfony server:start
```

When I open the homepage `http://localhost:8000/` I already have 2 errors in the debug toolbar : 
`Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://localhost:8000/"" at C:\Users\[user]\[project-path]\vendor\symfony\http-kernel\EventListener\RouterListener.php line 130`
and
`error while trying to collect executed migrations`

If I install the annotations dependency, I still have the errors.

Starting a complete new project with errors isn't a good start...

### How to reproduce

- Windows 11 Entreprise x64
- php : 8.1.8 (I tried with TS and NTS VS16 packages, I don't know which one should I use...)
- symfony CLI : 5.4.11

### Additional Context

I wanted to create a Symfony 6.1 project (current version apparently) but it seems impossible. When I tried : 
`symfony new my_project --version="6.1.*" --webapp`
An error occured : 
```
Could not find package symfony/skeleton with version 6.1.* in a version installable using your PHP version, PHP extensions and Composer version.
```

And when I tried (without specifying the version) : `symfony new my_project --webapp`
It installs a 6.0.10 project.

How to install the current 6.1 version of Symfony ?

**Comment by sandoba** [comment, marked as answer]
While these error messages can be reproduced this is basically expected behavior as there are additional steps necessary while setting up a new project (even if this is only partially mentioned in the default 404 screen).

Running
**symfony new test --webapp**
with
**Symfony CLI 5.4.12**
**PHP 8.1.7**
installs Symfony in version 6.1.2 at the moment.

To fix these initial errors:

1) Set up the database connection in "DATABASE_URL" in file ".env" (e.g. switch to SQLite instead of PostgreSQL for testing).
2) Create an initial controller, e.g.

symfony console make:controller
IndexController

In new file "/src/Controller/IndexController.php" replace
#[Route('/index', name: 'app_index')]
with
#[Route('/', name: 'app_index')]

The new controller action will show up at the root and both errors are now gone.
But indeed it is debatable if Symfony should default to a PostgreSQL setup while not having a default controller for the root.

**Comment by eved42** [comment]
Ok thank you @sandoba , it works. I think it's important to add it in the Symfony documentation because when you are a beginner and you start a new project with errors, it is disturbing. I'm used to using Symfony 3.4 and I wanted to try Symfony 6 and with older versions, I never had errors after creating a new project.
